app/discussion/client/views/DiscussionTabbar.html

```diff
@@ -1,24 +1,3 @@
 <template name="discussionsTabbar">
-	{{#if Template.subscriptionsReady}}
-		{{#unless hasMessages}}
-			<div class="list-view discussions-list flex-tab__header">
-					<h2>{{_ "No_discussions_yet"}}</h2>
-			</div>
-		{{/unless}}
-	{{/if}}
-	<div class="flex-tab__result discussions-list js-list">
-		<ul class="list clearfix">
-			{{# with messageContext}}
-				{{#each msg in messages}}
-					{{> message msg=msg room=room subscription=subscription groupable=false settings=settings u=u}}
-				{{/each}}
-			{{/with}}
-		</ul>
-
-		{{#if hasMore}}
-			<div class="load-more">
-				{{> loading}}
-			</div>
-		{{/if}}
-	</div>
+	{{ > DiscussionMessageList rid=rid onClose=close}}
```

Comment:
```suggestion
	{{> DiscussionMessageList rid=rid onClose=close}}
```

Small change to help grep-like searchs.